modules/sor/lib/sor.ts

```diff
@@ -30,6 +32,13 @@ export class SOR {
     ): Promise<PathWithAmount[] | null> {
         const checkedSwapAmount = checkInputs(tokenIn, tokenOut, swapKind, swapAmountEvm);
 
+        // get current block timestamp for ReClamm math
+        const currentTimestamp = await getCurrentBlockTimestamp(
+            tokenIn.chainId,
+            swapOptions?.rpcUrl,
+            swapOptions?.block,
+        );
```

Comment:
Could we just use `moment().now()` and have an option to pass a timestamp directly for tests?

Reply:
I also think we can actually get rid of `swapOptions?.block`?

Reply:
After some discussion on Slack we decided to try Date.now() - will update the PR and check how it behaves.
I'll try to get rid of block as well 👍